packages/ember-test-app/app/templates/faq.hbs

```diff
@@ -0,0 +1,15 @@
+{{page-title "FAQs"}}
+<main data-theme="marketing" class="bg-primary-subtle vh-100 vw-100">
+  <div class="container p-5">
+    <Mktg::Faq
+      @open={{true}}
+      @question="How does a FAQ Component work?"
+      @answer="Once you click the icon in the upper right corner, this answer will
```

Comment:
Since it's reasonable to think that the 'answer' section could include rich content such as links and images, could we move this into a `yield` block so it can be written with html?

Reply:
Are we comfortable with leaving the question as a parameter, or for consistency, should that be a yield block as well?

Reply:
I think leaving it as a parameter is probably fine.
Adding a second yield will increase complexity, and I don't expect to need complex formatting in the question.